doc/users/whats_new.rst

```diff
@@ -67,83 +116,86 @@ angular ticks have been modified to be parallel to the grid line. It may also
-path.
+    plt.savefig('test.png', metadata={'Software': 'My awesome software'})
 
 
-CheckButtons widget get_status function
----------------------------------------
+Busy Cursor
+-----------
 
-A :func:`~matplotlib.widgets.CheckButtons.get_status` method has been added the
-:class:`matplotlib.widgets.CheckButtons` class. This ``get_status`` method
-allows user to query the status (True/False) of all of the buttons in the
-``CheckButtons`` object.
+The interactive GUI backends will now change the cursor to busy when
+Matplotlib is rendering the canvas.
 
+PolygonSelector
+---------------
 
-Abstract base class for movie writers
--------------------------------------
+A :class:`~matplotlib.widgets.PolygonSelector` class has been added to
+:mod:`matplotlib.widgets`.  See
+:ref:`sphx_glr_gallery_widgets_polygon_selector_demo.py` for details.
 
-The new :class:`~matplotlib.animation.AbstractMovieWriter` class defines
-the API required by a class that is to be used as the `writer` in the
-`save` method of the :class:`~matplotlib.animation.Animation` class.
-The existing :class:`~matplotlib.animation.MovieWriter` class now derives
-from the new abstract base class.
 
+Added `matplotlib.ticker.PercentFormatter`
+------------------------------------------
 
-Add fill_bar argument to ``AnchoredSizeBar``
---------------------------------------------
+The new `~matplotlib.ticker.PercentFormatter` formatter has some nice
+features like being able to convert from arbitrary data scales to
+percents, a customizable percent symbol and either automatic or manual
+control over the decimal points.
 
-The ``mpl_toolkits`` class
-:class:`~mpl_toolkits.axes_grid1.anchored_artists.AnchoredSizeBar` now has an
-additional ``fill_bar`` argument, which makes the size bar a solid rectangle
-instead of just drawing the border of the rectangle. The default is ``None``,
-and whether or not the bar will be filled by default depends on the value of
-``size_vertical``. If ``size_vertical`` is nonzero, ``fill_bar`` will be set to
-``True``. If ``size_vertical`` is zero then ``fill_bar`` will be set to
-``False``. If you wish to override this default behavior, set ``fill_bar`` to
-``True`` or ``False`` to unconditionally always or never use a filled patch
-rectangle for the size bar.
 
+Reproducible PS, PDF and SVG output
+-----------------------------------
 
-Annotation can use a default arrow style
-----------------------------------------
+The ``SOURCE_DATE_EPOCH`` environment variable can now be used to set
+the timestamp value in the PS and PDF outputs. See `source date epoch
+<https://reproducible-builds.org/specs/source-date-epoch/>`__.
 
-Annotations now use the default arrow style when setting ``arrowprops={}``,
-rather than no arrow (the new behavior actually matches the documentation).
+Alternatively, calling ``savefig`` with ``metadata={'creationDate': None}``
+will omit the timestamp altogether.
```

Comment:
This is not consistent with the above docs on metadata which say that "Creator" is the only supported key for (e)ps.

Reply:
Good catch, this only applies to the pdf backend.